.github/workflows/ci-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI Workflow 
+on: [push, pull_request]
+jobs:
+  build:
+    name: build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [11.x]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with: 
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm ci
```

Comment:
I don't know much about JavaScript stuff, but I imagine the commands used here should match  what's in https://github.com/freeCodeCamp/chapter/blob/93f6aad27c81a86f655f27ca63d9069a442d06ff/package.json#L9-L23

Reply:
@ScottBrenner `npm ci` install a project with a clean slate. For more info: https://docs.npmjs.com/cli/ci.html.